vcpkg.json

```diff
@@ -1,6 +1,6 @@
 {
-  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",

```

Comment:
why was the schema removed? it provides context for code editors on what this file is supposed to contain which is important for validation